yarn.lock

```diff
@@ -21739,6 +21739,18 @@ object-path-immutable@^3.1.1:
   dependencies:
     is-plain-object "3.0.0"
 
+object-prototype-functions@^1.2.0:
+  version "1.2.0"
+  resolved "https://registry.yarnpkg.com/object-prototype-functions/-/object-prototype-functions-1.2.0.tgz#59ffa520f2e7ef3b3ba427ba81178eb8feca1048"
+  integrity sha512-7pfkHi5KHFz/B3RhQVORykDYF2CdQY/h1isJzCJJfEubKkSO59io1raYHAMUrx0Im3fUSrU1oX6E4KeCwyU/jQ==
+
+object-prototype@^2.0.0:
+  version "2.0.0"
```

Comment:
Hmm good call - I'll see what I can do...

Reply:
Progress update: So far I've not been able to do anything about this that doesn't expose `Object.prototype`, so I don't have high hopes for solving this. I'm still investigating though...

Reply:
Do you think it's possible to take advantage of https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy to prevent the look-up of properties from the root prototypes of `Object`, `Function`, etc. that don't match what we expect?

Reply:
Fixed... not using Proxy, but by creating a fake `FunctionPrototype` similar to the `ObjectPrototype`